packages/simulation-new/src/workshop/protocol/protocol.ts

```diff
@@ -36,10 +36,62 @@ export namespace Events {
   ] as const;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-namespace
+export namespace Queries {
```

Comment:
Not sure about using a namespace here, maybe we should have this in its own file?